tests/e2e/Services/Account/AccountCustomClientTest.php

```diff
@@ -480,6 +480,20 @@ public function testUpdateAccountPassword($data): array
         $password = $data['password'] ?? '';
         $session = $data['session'] ?? '';
 
+        for ($i = 0; $i < 5; $i++) {
+            $response = $this->client->call(Client::METHOD_POST, '/account/sessions/email', array_merge([
+                'origin' => 'http://localhost',
+                'content-type' => 'application/json',
+                'x-appwrite-project' => $this->getProject()['$id'],
+            ]), [
+                'email' => $email,
+                'password' => $password,
+            ]);
+
+            $this->assertEquals(201, $response['headers']['status-code']);
+            sleep(1);
+        }
```

Comment:
This should fail 🤔 you can't create a session (in the same client) when you already have an active one